src/main/webapp/form.html

```diff
@@ -7,15 +7,16 @@
     <link rel="stylesheet" href="/styles/fonts.css">
     <link rel="stylesheet" href="/styles/common.css">
     <link rel="stylesheet" href="/styles/form.css">
+    <script src="/scripts/form.js"></script>
   </head>
-  <body>
+  <body onload="loadUploadUrl()">
```

Comment:
nit: this name seems a little confusing, is there anything you could clarify by changing it?

Reply:
this is to load the url for uploading data from the form, do you have any name suggestion